library/src/main/java/io/github/luizgrp/sectionedrecyclerviewadapter/SectionedRecyclerViewAdapter.java

```diff
@@ -413,6 +444,33 @@ public int getSectionPosition(String tag) {
         return sections;
     }
 
+
+    /**
+     * Helper method that receives position in relation to the section, and returns the position in
+     * the adapter.
+     *
+     * @param tag unique identifier of the section
+     * @param position position of the item in the section
+     * @return position of the item in the adapter
+     */
+    public int getAdapterPosition(String tag, int position) {
```

Comment:
Could you please open a separate PR for these two `getAdapterPosition` methods? Please rename it to `getPositionInAdapter` to make it consistent with the other method named `getPositionInSection`.